charts/brigade/templates/artemis/secondary-stateful-set.yaml

```diff
@@ -88,6 +88,11 @@ spec:
             port: core
           initialDelaySeconds: 10
           periodSeconds: 10
+        startupProbe:
+          tcpSocket:
+            port: core
+          failureThreshold: 10
+          periodSeconds: 10
```

Comment:
Same comment as [here](https://github.com/brigadecore/brigade/pull/1304/files#r585682226)

Reply:
Sorry... the comment here was really about needing a higher failure threshold.